docs/getting-started.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started

```

Comment:
"Getting Started" as in local workspace setup (if not using the gitpod environment) for contributing should be on the main README. this additional doc here should only cover project structure details that would be too in-depth for the readme.

Reply:
Moved the whole setup documentation to README and renamed `getting-started.md` to `project-structure.md`.